mithril-common/src/entities/cardano_database.rs

```diff
@@ -0,0 +1,83 @@
+use semver::Version;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    entities::{CardanoDbBeacon, CompressionAlgorithm},
+    signable_builder::Artifact,
+};
+
+/// Cardano database incremental.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct CardanoDatabase {
```

Comment:
This name is not specific enough:
```suggestion
/// Cardano database snapshot.
#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
pub struct CardanoDatabaseSnapshot {
```

Reply:
Done ✅